src/shared/Form/LabelText.tsx

```diff
@@ -4,17 +4,26 @@ import { REQUIRED_INPUT } from '../../config';
 import { RequiredInputLabel, SecondaryInfoText } from './';
 
 interface ILabelTextProps {
+  // Label text
   label: any;
+
+  // Is this a required field
   required?: boolean;
+
+  // Should this field display a * to let user know it's required
+  // (a field can be required, but still have showRequiredLabel set to false)
```

Comment:
Why do we want this scenario?

Reply:
So we can have required fields that don't show the required star (e.g. if all form fields are understood to be required)